src/test/kotlin/cacophony/semantic/analysis/StaticFunctionRelationsTest.kt

```diff
@@ -320,18 +321,18 @@ class StaticFunctionRelationsTest {
     fun `should find relations in complex nested functions`() {
         // given
         // (foo => (let a; g => h => a; i => (j => (); g())); main => foo())
-        val varA = variableDeclaration("a", Empty(mockRange()))
-        val varAUse = variableUse("a")
-        val funH = functionDeclaration("h", varAUse)
-        val funG = functionDeclaration("g", funH)
-        val funJ = functionDeclaration("j", block())
-        val varGUse = variableUse("g")
-        val funI = functionDeclaration("i", block(funJ, call(varGUse)))
-        val funFoo = functionDeclaration("foo", block(varA, funG, funI))
-        val varFooUse = variableUse("foo")
-        val funMain = functionDeclaration("main", call(varFooUse))
-
-        val ast = astOf(funFoo, funMain)
+        val varA = cacophony.semantic.variableDeclaration("a", Empty(cacophony.semantic.mockRange()))
```

Comment:
Is this verbosity necessary?

Reply:
It was done automatically and I did not notice it - i could either import all of these mock classes or move TestUtils to package cacophony.semantic.analysis package, as they are used almost only in here, but I'm not sure if this will be intuitive